### ISSUE TITLE
When mute or unmute microphone during ongoing call it rebuilding custom audioVideoView

### ISSUE THREAD

**Issue by atulproject99** [ZEGOCLOUD/zego_uikit_prebuilt_call_flutter#67]
- [x] I have read the [Getting Started](https://www.zegocloud.com/doc/callkit/Call%20Kit/main/Quick%20start/b26d509e) and [Getting Started(with call invitation)](https://www.zegocloud.com/doc/callkit/Call%20Kit/main/Quick%20start%20(with%20call%20invitation)/eb1d3c42)
- [x] I have already searched for the same problem

## Environment

| Technology            | Version       |
|-----------------------| ------------- |
| Flutter version       |  Latest             |
| Plugin version        |   Latest            |
| Android version       |               |
| iOS version           |               |
| macOS version         |               |
| Xcode version         |               |
| Google Chrome version |               |

Device information: 

## Description

**Expected behavior:** 
During the onoing call mute or unmute shouldn't impact on audioVideo view.

**Current behavior:** 
When mute or unmute microphone during ongoing call it rebuilding custom audioVideoView
## Steps to reproduce

1. This
2. Than that
3. Then

**Comment by atulproject99** [comment]
<img width="1099" alt="Screenshot 2024-09-26 at 11 36 55 AM" src="https://github.com/user-attachments/assets/cc8ed2f9-7585-4009-a50f-19bb215a95bf">

mute or unmute during call rebuild every time audioVideoView in both side @yoer please help with this issue

**Comment by yoer** [comment]
mute or unmute, there will be user microphone status attribute updates, so it is right for containerBuilder to trigger update.

**Comment by yoer** [comment]
This should only involve building the interface. 

Did you write any time-consuming business-related code?

If so, I suggest you cache your own data

**Comment by atulproject99** [comment]
yes @yoer i have writting some other business related code i will check it based on your suggestion
